dGame/dBehaviors/AreaOfEffectBehavior.cpp

```diff
@@ -20,134 +20,123 @@ void AreaOfEffectBehavior::Handle(BehaviorContext* context, RakNet::BitStream* b
 		return;
 	}
 
+	if (this->m_useTargetPosition){
+		Game::logger->Log("AreaOfEffectBehavior", "target is %llu", branch.target);
+		if (branch.target == LWOOBJID_EMPTY)
+		return;
+	}
+
+	if (targetCount == 0){
+		Game::logger->Log("AreaOfEffectBehavior", "no targets");
+		PlayFx(u"miss", context->originator);
```

Comment:
Now thinking about it, the client handles what FX to play in a handle.  Possibly in calculate too.  Should double check this to make sure we arent doubling up on FX

Reply:
I did check this and the FX were not happening before these changes